statsmodels/datasets/tests/test_utils.py

```diff
@@ -16,11 +16,11 @@ def test_get_rdataset():
     test_url = "https://raw.githubusercontent.com/vincentarelbundock/" \
                "Rdatasets/master/csv/datasets/cars.csv"
     internet_available = check_internet(test_url)
-    if not internet_available:
+    if not internet_available:  # pragma: no cover
         pytest.skip('Unable to retrieve file - skipping test')
     try:
         duncan = get_rdataset("Duncan", "carData", cache=cur_dir)
-    except (HTTPError, URLError):
+    except (HTTPError, URLError):  # pragma: no cover
```

Comment:
These are already in coveragerc

Reply:
https://coveralls.io/builds/23535701/source?filename=statsmodels/datasets/tests/test_utils.py

Looks like its not in coveragerc_travis.  I'll update it there.  I missed the discussion that led to having multiple coveragerc files

Reply:
If we could get back to just one coveragerc file, we could drop that and put it into setup.cfg